examples/lighting-app/efr32/args.gni

```diff
@@ -22,3 +22,4 @@ pw_log_BACKEND = "${chip_root}/src/lib/support/pw_log_chip"
 pw_assert_BACKEND = "$dir_pw_assert_log"
 chip_enable_openthread = true
 chip_openthread_ftd = true
+chip_system_config_use_ot_udp = true
```

Comment:
even if longer, lets use `open_thread` instead of `ot` to optimize for readability instead of typing speed:

```
chip_system_config_use_open_thread_udp = true
```

Reply:
Done